README.md

```diff
@@ -98,4 +105,79 @@ automatically remove set all auto-merge labels from the PR
 _Note_: at the moment the Dlang-Bot doesn't store _any_ authentication token, so
 it can't perform actions on behalf of others.
 
-<a name="canceling-state" />
+<a name="warn-common-mistakes" />
```

Comment:
Hehe thanks. I am aware of them, but the current solution works well and imho  isn't a big overhead. I would be very happy if the Dlang-Bot ever gained so many features that we need to opt for such an approach.